libbeat/scripts/generate_template.py

```diff
@@ -292,12 +293,12 @@ def fill_field_properties(args, field, defaults, path):
             path = field["name"]
         prop, dynamic = fill_section_properties(args, field, defaults, path)
 
+        properties[field.get("name")] = {
+            "type": "nested",
```

Comment:
Actually you are right. The default is 'object' so that 'nested' is required here.